terraform/hub/cw-dashboard-vpc-cni.json

```diff
@@ -118,13 +118,13 @@
                 "view": "timeSeries",
                 "stacked": false,
                 "metrics": [
-                    [ "Kubernetes", "eniAllocated", "CLUSTER_ID", "${CLUSTER_ID}" ],
-                    [ "Kubernetes", "eniMaxAvailable", "CLUSTER_ID", "${CLUSTER_ID}" ]
+                    [{ "expression": "SUM(SEARCH('{Kubernetes,CLUSTER_ID} MetricName=\"eniAllocated\" CLUSTER_ID=\"**default_cluster**\"', 'Sum', 300))", "id": "m1" }],
+                    [{ "expression": "SUM(SEARCH('{Kubernetes,CLUSTER_ID} MetricName=\"eniMaxAvailable\" CLUSTER_ID=\"**default_cluster**\"', 'Sum', 300))", "id": "m2" }]
                 ],
                 "region": "**aws_region**",
                 "title": "ENIs",
                 "period": 300
             }
         }
     ]
-}
\ No newline at end of file
+}
```

Comment:
IP addresses and ENI numbers seem to be multiplied by 10. Metrics names are incorrect e.g. m1.